src/frontend/src/components/SearchPage/SearchPage.jsx

```diff
@@ -26,10 +26,15 @@ const useStyles = makeStyles(() => ({
 const SearchPage = () => {
   const { telescopeUrl } = useSiteMetadata();
   const classes = useStyles();
-
-  const [searchText, setSearchText] = useState('');
+  let [keyword] = useQueryParam('q', StringParam);
```

Comment:
Probably this should be `text` or `searchText`

Reply:
You can set your defaults for these two values like this:

```js
const [searchText = ''] = useQueryParam('text', StringParam);
const [filter = 'post'] = useQueryParam('filter', StringParam);
```

Reply:
I can do the initial like this?  Learned it. Thank you so much.

Reply:
I had to look it up too.  I was surprised that `useQueryParam()` doesn't provide a mechanism to do it.  This is using what modern JS provides vs. what React/useQueryParam give.

Reply:
It is really helpful. I spent a lot of time last night trying to make my code streamline and effective. If you didn't mention it, I could never know this approach.